core/src/rpc_pubsub_service.rs

```diff
@@ -44,7 +45,7 @@ impl PubSubService {
                         session
                 })
                 .max_connections(1000) // Arbitrary, default of 100 is too low
-                .max_payload(10 * 1024 * 1024 + 1024) // max account size (10MB) + extra (1K)
+                .max_payload(MAX_REQUEST_PAYLOAD_SIZE)
```

Comment:
This actually only influences the request payload size

https://github.com/paritytech/jsonrpc/blob/master/ws/src/server_builder.rs#L111-L133

https://github.com/paritytech/jsonrpc/blob/master/ws/src/server.rs#L69-L70

https://github.com/housleyjk/ws-rs/blob/master/src/lib.rs#L161-L163

Reply:
@t-nelson  It also affects the output buffer size: https://github.com/paritytech/jsonrpc/blob/master/ws/src/server.rs#L72

Reply:
Ah I see, the line I just linked above was changed here: https://github.com/paritytech/jsonrpc/commit/92e3c14a1ebc1f3dc066e9721ffcd95b8f7cbc3f

Reply:
Back to the previous `.max_payload()` for now?

Reply:
Sounds good, I created https://github.com/paritytech/jsonrpc/issues/590 as well

Reply:
PR here: https://github.com/solana-labs/solana/pull/12548